kolibri/utils/tests/test_main.py

```diff
@@ -105,13 +106,19 @@ def test_conditional_backup():
     dbbackup("0.13.4")
     dbbackup("0.13.5")
 
+    # set the version to the current version + 1
+    version = truncate_version(kolibri.__version__)
+    higher_version = [str(int(x) + 1 for x in version.split("."))]
```

Comment:
Making the generator into a string, rather than the value.
```suggestion
    higher_version = [str(int(x) + 1) for x in version.split(".")]
```